### PR TITLE
Config route

### PR DESCRIPTION
**Pull request by runnersaw**
Refactors pm.py for 
- Static methods that take identifiers rather than instance creation
- Updating config
- Only keeping one copy of tracking.cfg

Adds config route that
- Updates tracking.cfg

Untested, and can't really be until Deniz's changes get merged. I need his stuff to be done in order to hook up his uploadVideo and uploadHomography routes to my pm changes.

### CODE REVIEW

File: trafficcloud/api.py
Comment (runnersaw) [inline trafficcloud/api.py:14]: Note that this will break the current workflow until Deniz's changes get merged.

**Review by jabb1123 — COMMENTED** [review]
From what I could tell, looked Good. I'm holding off on approval until @dcelik can ok the changes here as well.

File: app/handlers/config.py
Comment (jabb1123) [inline app/handlers/config.py:5]: Perhaps we should move the pm.py file to traffic)cloud)utils now? It seems like we can remove the old trafficcloud folder if we do so.

File: trafficcloud/api.py
Comment (jabb1123) [reply trafficcloud/api.py:14]: Isn't this file fairly superfluous at this point? Why not move all these function under the handlers that call them? Then, in the future, if we need to change them, we can edit the handlers that now have these functions. In addition, we can remove this file, and in turn, this whole trafficcloud folder.

**Review by jabb1123 — CHANGES_REQUESTED** [review]

File: app/handlers/config.py
Comment (dcelik) [reply app/handlers/config.py:5]: this can be done post-merge, during refactoring

File: trafficcloud/api.py
Comment (dcelik) [reply trafficcloud/api.py:14]: Why does project wizard no longer generate a uuid?? This identifier creation will be added to UploadVideoHandler.

**Review by dcelik — APPROVED** [review]

File: trafficcloud/api.py
Comment (dcelik) [reply trafficcloud/api.py:14]: This will require significant changes to uploadVideo handler to function correctly. Will not be merged until my changes can be merged.

**Review by dcelik — CHANGES_REQUESTED** [review]

**Pull request merged**
